resources/apigateway-domainnames.go

```diff
@@ -43,10 +43,25 @@ func (l *APIGatewayDomainNameLister) List(ctx context.Context, o interface{}) ([
 
 		for i := range output.Items {
 			item := &output.Items[i]
+			
+			// Get tags for the domain
+			tagsOutput, err := svc.GetTags(ctx, &apigateway.GetTagsInput{
+				ResourceArn: aws.String("arn:aws:apigateway:" + opts.Config.Region + "::/domainnames/" + *item.DomainName),
+			})
+			if err != nil {
+				return nil, err
```

Comment:
Please do not return an error here, instead we need to log that an error has occurred and allow the for loop to continue down to adding the resource, we don't block adding the resource if tags are unavailable.